libqtile/widget/base.py

```diff
@@ -93,9 +93,19 @@ class _Widget(CommandObject, configurable.Configurable):
     have been configured.
 
     Callback functions can be assigned to button presses by passing a dict to the
-    'callbacks' kwarg. For example: {'Button1': func} will execute func when the widget
-    receives a button 1 press. The Qtile instance of passed as the only argument to the
-    callback functions.
+    'callbacks' kwarg.
+
+    For example::
```

Comment:
I'm not a rST expert but we'll miss Python syntax highlighting, nah?

```suggestion
    For example:

    .. code-block:: python
```

Reply:
I wasn't sure either so I coped it from the Columns layout because I remembered it had a config example too. Looks like you're right, and there are a few other instances of the first form in the dostrings.